server/src/main/java/org/opensearch/search/profile/query/CollectorResult.java

```diff
@@ -88,15 +94,59 @@ public class CollectorResult implements ToXContentObject, Writeable {
      */
     private final Long time;
 
+    /**
+     * The total elapsed time in reduce phase for this CollectorManager
+     */
+    private final Long reduceTime;
```

Comment:
I realize `time` above uses a Long type, but a quick glance through the code suggests these fields can never be set to null. Can you use primitive types (like you did for `sliceCount`)?

Reply:
Thank you, @andrross for the review. I have now made the necessary adjustments.